google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/RemoveFeedItemAttributeValue.java

```diff
@@ -0,0 +1,261 @@
+    updateFeedItem(googleAdsClient, params.customerId, params.feedId, params.feedItemId);
+  }
+
+  /**
+   * Removes the flight price from the feed item. In order to remove a FeedItemAttributeValue you
+   * must update the FeedItem.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID in which to create criterion.
+   * @param feedId the ID of the feed on which the feedItemAttribute value will be removed.
+   * @param feedItemId the ID of the feed item on which the feedItemAttribute value will be removed.
+   */
+  private void updateFeedItem(
+      GoogleAdsClient googleAdsClient, long customerId, long feedId, long feedItemId) {
+    // Gets the feed resource name.
+    String feedResourceName = ResourceNames.feed(customerId, feedId);
+
+    // Gets a map of the placeholder values and feed attributes.
+    Map<FlightPlaceholderField, FeedAttribute> feedAttributes =
+        AddFlightsFeed.getFeed(googleAdsClient, customerId, feedResourceName);
+
+    // Gets the feed item resource name.
+    String feedItemResourceName = ResourceNames.feedItem(customerId, feedId, feedItemId);
+    // Removes the attribute from the feed item.
+    FeedItem feedItem =
+        removeAttributeValueFromFeedItem(
+            googleAdsClient,
+            customerId,
+            feedAttributes,
+            feedItemResourceName,
+            FlightPlaceholderField.FLIGHT_SALE_PRICE);
+
+    // Creates the operation.
+    FeedItemOperation operation =
+        FeedItemOperation.newBuilder()
+            .setUpdate(feedItem)
+            .setUpdateMask(FieldMasks.allSetFieldsOf(feedItem))
+            .build();
+
+    // Creates the feed item service client.
+    try (FeedItemServiceClient feedItemServiceClient =
+        googleAdsClient.getLatestVersion().createFeedItemServiceClient()) {
+      // Updates the feed item.
+      MutateFeedItemsResponse response =
+          feedItemServiceClient.mutateFeedItems(
+              Long.toString(customerId), ImmutableList.of(operation));
+      for (MutateFeedItemResult result : response.getResultsList()) {
+        System.out.printf("Updated feed item with resource name '%s'.%n", result.getResourceName());
+      }
+    }
+  }
+
+  /**
+   * Removes a feed item attribute value.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID in which to create criterion.
+   * @param feedItemResourceName the resource name of the feedItem to be updated.
+   * @param flightPlaceholderField the placeholder type for the attribute to be removed.
+   */
+  private FeedItem removeAttributeValueFromFeedItem(
+      GoogleAdsClient googleAdsClient,
+      long customerId,
+      Map<FlightPlaceholderField, FeedAttribute> feedAttributes,
+      String feedItemResourceName,
+      FlightPlaceholderField flightPlaceholderField) {
+    // Gets the ID of the placeholder attribute field and converts to an integer.
```

Comment:
nit: "placeholder attribute field" is a bit confusing. How about just "FeedAttribute for the placeholder field"?